index.html

```diff
@@ -7,37 +7,31 @@
 	<link rel="stylesheet" href="quiz.css">
 </head>
 <body>
-	
-<article class="item">
-
-<!-- This is the first section of the card -->
 
+<article class="item">
 <section class="item">
-	<div class= "div1">
+	<div class= "section1">
 	<h1>Dominion Card #1: Kids these Days</h1>
 	</div>
```

Comment:
I would avoid creating classes like "div1" "div2" "section1" "section2". Try to make them more semantic... "product-title" "product-image" "product-description" "product-availability" "product-specifications" "product-pricing" add much more clarity.